students/James Shimshak/Lab4_FinalCode/Lab4_FinalCode.c

```diff
@@ -0,0 +1,139 @@
+#include <avr/io.h>
+#include <util/delay.h>
+#define LED_DDR_L               (DDRD)
+#define LED_PORT_L              PORTD
+#define DELAYTIME 85
+#define LED_DDR_R               (DDRD)
```

Comment:
It looks like your two directions are using the same registers. (e.g. `LED_DDR_L` is the same as `LED_DDR_R`)